rasa/graph_components/adders/nlu_prediction_to_history_adder.py

```diff
@@ -33,9 +33,9 @@ def create(
     def add(
         self,
         predictions: List[Message],
-        tracker: DialogueStateTracker,
-        domain: Domain,
+        tracker: Optional[DialogueStateTracker],
         original_messages: List[UserMessage],
+        domain: Optional[Domain] = None,
```

Comment:
These are now optional because we want to have the message added to the tracker for nlu-only models.